Changelog.md

```diff
@@ -14,6 +19,12 @@
   constraint.
 * Change the associativity of combinators `serial`, `wSerial`,
   `ahead`, `async`, `wAsync`, `parallel` to be the same as `<>`.
+* `encodeUtf8`, `decodeUtf8` now replace any invalid character encountered
+  during encoding/decoding with the Unicode replacement character. Use
+  `encodeUtf8'`, `decodeUtf8'` to recover the previous functionality.
+* `encodeLatin1` now silently truncates any character beyond 255 to incorrect
+  characters in the input stream. Use `encodeLatin1'` to recover previous
+  functionality.
```

Comment:
Since these routines are in a new module, they are not technically breaking changes in themselves. We can write in the `Streamly.Data.Unicode.Stream` deprecation comment above that the routines in the new module have different behavior.